operator/cmd/mesh/manifest-apply.go

```diff
@@ -108,7 +121,7 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
 //  verbose full manifests are output
 //  wait    block until Services and Deployments are ready, or timeout after waitTimeout
 func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRun bool, verbose bool,
-	kubeConfigPath string, context string, wait bool, waitTimeout time.Duration, l *Logger) error {
+	kubeConfigPath string, certDir string, context string, wait bool, waitTimeout time.Duration, l *Logger) error {
```

Comment:
This starts to look like it needs a struct or context object... I guess it was already looking that way before with 9 parameters.